src/main/java/racingcar/model/RacingGame.java

```diff
@@ -0,0 +1,42 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+    private final List<Car> cars;
+    private final int rounds;
+    private int currentRound;
+
+    public RacingGame(List<Car> cars, int rounds) {
+        this.cars = cars;
+        this.rounds = rounds;
+        this.currentRound = 0;
+    }
+
+    public boolean hasNextRound() {
+        return currentRound < rounds;
+    }
+
+    public void playRound() {
+        for (Car car : cars) {
+            car.move();
+        }
+        currentRound++;
+    }
+
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    public List<String> getWinners() {
+        int maxPosition = cars.stream().mapToInt(Car::getPosition).max().orElse(0);
```

Comment:
```suggestion
        int maxPosition = cars.stream()
                              .mapToInt(Car::getPosition)
                              .max()
                              .orElse(0);
```

이렇게 스트림 체이닝을 사용하시면 더 가독성이 높아질 것 같습니다!